site/content/release-notes/3.7.5.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "3.7.5"
+toc: true
+---
+RDF4J 3.7.5 is a patch release that fixes 14 issues.
+
+For a complete overview, see [all issues fixed in 3.7.5](https://github.com/eclipse/rdf4j/milestone/74?closed=1).
+
+## Acknowledgements
+
+This release was made possible by contributions from ???
```

Comment:
There's quite a few names of contributors to add here. I typically fill this out by having a look at the [closed issues in the milestone](https://github.com/eclipse/rdf4j/issues?q=is%3Aclosed+milestone%3A3.7.5) and looking at the assignees for each ticket.

Reply:
Yep, this is just the boilerplate that the scripts generate. First time running the scripts on my new mac and had some issues with tools I needed to install and some bugs I had to fix to get it all working. That took some time to get sorted so this was how far I got today. Tomorrow I'll update the release notes and merge/publish them.